File: Online_Judge_Problems/BOJ/boj_9086/boj_9086.js
/*
----------------------------------------------
Sub : [BOJ] 문자열
Link: https://www.acmicpc.net/problem/9086
Tag : js, 문자열
Memo
----------------------------------------------
*/

// fs모듈을 통한 입력
const fs = require("fs");
const input = fs.readFileSync("/dev/stdin").toString().trim().split("\n");

// 테스트 케이스의 개수 입력
const T = Number(input[0]);
// 문자열 배열 선언
let s = [];

for (let i=1; i<=T; i++) {
  s.push(input[i])
}
for (let i=0; i<T; i++) {
  // 첫 글자와 마지막 글자를 추출하여 출력
  console.log(s[i][0] + s[i][s[i].length -1])
}

File: Online_Judge_Problems/BOJ/boj_9086/boj_9086.c
/*
----------------------------------------------
Sub : [BOJ] 문자열
Link: https://www.acmicpc.net/problem/9086
Tag : C, 문자열
Memo
----------------------------------------------
*/

#include <stdio.h>
#include <string.h>

int main() {
  int T;           // 테스트 케이스의 개수
  scanf("%d", &T); // 입력받기

  for (int i = 0; i < T; i++) {
    char s[1001];
    scanf("%s", s);

    int len = strlen(s);
    printf("%c%c\n", s[0], s[len - 1]); // 첫 글자와 마지막 글자 출력
  }

  return 0;
}
